app/config/config.yml

```diff
@@ -11,6 +11,13 @@ imports:
 
 # Basic configuration for the Symfony framework features
 framework:
+    # Uncomment the 'ide' option to turn file paths into clickable links that
+    # open the given file path and line number using your favorite IDE.
+    # Supported values are 'textmate', 'macvim', 'emacs' and 'sublime'
+    # See http://symfony.com/doc/current/reference/configuration/framework.html#ide
+    #
+    # ide: sublime
```

Comment:
@aitboudad why in config_dev ? It could be set for any environment without issue.

However, I would indeed recomment setting ``xdebug.file_link_format`` whenever possible. this allows configuring it only once for any project on your machine, and it makes things easier if people in your team are not using the same IDE